test/unit/view.spec.js

```diff
@@ -151,8 +131,8 @@ describe('item view', function() {
       expect(this.attachElContentSpy).to.not.have.been.called;
     });
 
-    it('should claim isRendered', function() {
-      expect(this.view.isRendered()).to.be.true;
+    it('should not claim isRendered', function() {
```

Comment:
I think we should test also a template: false with a prerendered content. AFAIK isRendered() should be true when the view is created on an el with content

Reply:
Agreed.  Here's an example of that test setup: https://github.com/marionettejs/backbone.marionette/blob/master/test/unit/view.spec.js#L14
You can probably do this without too much extra setup by passing an `el` at instantiation like:
`this.elView = new this.View({ el: '#foo' });`
you just have to make sure `#foo` is in the fixture with content